### PR TITLE
Fixes #2016: Increases jest e2e timeout seconds

### LINKED ISSUE #2016: Investigate timeouts in e2e tests running in CI
[Sometimes the e2e tests will fail to run](https://github.com/Seneca-CDOT/telescope/runs/2177747320?check_suite_focus=true):

```
> jest -c jest.config.e2e.js --


Jest dev-server output:
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Error: Server has taken more than 5000ms to start.
    at setupJestServer (/home/runner/work/telescope/telescope/node_modules/jest-process-manager/lib/index.js:211:19)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 0)
    at async Object.setup (/home/runner/work/telescope/telescope/node_modules/jest-process-manager/lib/index.js:138:5)
    at async setup (/home/runner/work/telescope/telescope/node_modules/jest-playwright-preset/lib/global.js:25:13)
    at async /home/runner/work/telescope/telescope/node_modules/@jest/core/build/runGlobalHook.js:82:11
    at async waitForPromiseWithCleanup (/home/runner/work/telescope/telescope/node_modules/@jest/transform/build/ScriptTransformer.js:207:5)
    at async /home/runner/work/telescope/telescope/node_modules/@jest/core/build/runGlobalHook.js:72:9
    at async pEachSeries (/home/runner/work/telescope/telescope/node_modules/p-each-series/index.js:8:23)
npm ERR! code ELIFECYCLE
```

We might need to increase the 5000ms startup window so it works on the GitHub builders (I bet they are under-powered).

### PR DESCRIPTION
**Pull request by HyperTHD**
## Issue This PR Addresses
This PR closes #2016 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

Our e2e tests were failing due to the github actions vm possibly not having enough time to run the e2e tests. This PR adds an additional 5000 seconds to a total of 10000 milliseconds to run the e2e tests. Since the issue is in the e2e tests ran during a push or PR, we'll know if it works by looking at the e2e tests when this PR is made or whenever it gets updated/rebased. 

I started with 10000 just to be safe, and can easily increment this if the time is not enough.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — CHANGES_REQUESTED** [review]
You have 10 commits in here for what should be a single-line change to one file.  The change to SearchResults.tsx shouldn't be in there.  Can you please fix your history?

Also `100,000` is pretty long, I'd try `30,000` to start.

File: jest-playwright.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   serverOptions: {
     command: 'npx http-server src/api/auth/test/e2e -p 8888',
     port: 8888,
-    launchTime: 10000,
+    launchTime: 30000,
```
Comment (humphd) [inline jest-playwright.config.js:11]: `launchTimeout` is what it needs, I think vs. `launchTime`.

**Review by humphd — CHANGES_REQUESTED** [review]
Still seeing 10 commits.

**Comment by birtony** [comment]
@HyperTHD  try the following:
1. Check if you have your upstream configured correctly pointing to `git@github.com:Seneca-CDOT/telescope.git`: `git remote -v`
2. Get the latest code from the main telescope repo and reset your master branch, then check git history:
```bash
git checkout master
git fetch upstream
git reset --hard upstream/master
git log
```
2a. At this point, you may also want to push the updated master branch to your forked repo to update it there as well: `git push -f origin master` (optional)
3. Create a temporary branch and `cherry-pick` commits needed (I think you only have one for that PR - `e474969bb57d4f9368ebad7cd9a06fd9058dfcfe`). If you need to look up hashes of commits that you need to keep, checkout to `issue-2016` branch and see `git log` or use https://github.com/Seneca-CDOT/telescope/pull/2059/commits. Note that you need to be on the `tmp-branch` when you do a cherry-pick.
```bash
git checkout -b tmp-branch
git cherry-pick e474969bb57d4f9368ebad7cd9a06fd9058dfcfe <any-additional-hashes-space-separated>
```
4. Check git history and make sure all of your changes for this PR are there: `git log`
5. Remove branch issue-2016 locally: `git branch -D issue-2016`
6. Rename tmp-branch to issue-2016: `git branch -m "issue-2016"`
7. Force push to remote: `git push -f origin issue-2016`

**Comment by humphd** [comment]
Please fix your git history:

```
git checkout master
git pull upstream master
git diff master jest-playwright.config.js jest.config.e2e.js > 2059.diff
git checkout -B issue-2016 master
git apply 2059.diff
```

**Comment by humphd** [comment]
This got closed and never fixed.  I still need https://github.com/Seneca-CDOT/telescope/blob/master/jest-playwright.config.js#L11 to get updated.

**Comment by HyperTHD** [comment]
Another PR is coming up rn to close this issue, accidently closed this in my process of fixing the git history so made a brand new branch and added the changes there

**Pull request closed without merging**